Flipkart-price-alert/README.md

```diff
@@ -0,0 +1,25 @@
+## Flipkart Price Alert
+Checks for the price of a desired product periodically.
+If the price drops below the desired amount, notifies the user via email.
+## Setup Instructions
+Type the following to install requirements: (make sure you have python installed)
+`pip install -r requirements.txt`
+# How to get app password
```

Comment:
Not all of us store passwords in Google account. You can remove this section

Reply:
the smtp library requires the user's mail credentials, this details the way in which one can get the said credentials but for a limited scope (ie. just for accessing mail)